lib/gui/app/utils/middle-ellipsis/filter.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+/**
+ * The purpose of this module is to provide utilities
+ * to work with sizes in bytes.
```

Comment:
Typo? ;-)

Reply:
Copy-paste from the package – we're just inlining its contents in our codebase  – thanks for noticing!

Reply:
> we're just inlining its contents in our codebase

Does the original author need to be credited?

Reply:
I didn't really think about that since it's @jviotti's package and he approved, but that's a good call.